nilearn/masking.py

```diff
@@ -568,13 +570,16 @@ def compute_gray_matter_mask(target_img, threshold=.5,
     """
     return compute_brain_mask(target_img=target_img, threshold=threshold,
                               connected=connected, opening=opening,
-                              memory=memory, verbose=verbose)
+                              memory=memory, verbose=verbose,
+                              mask_type='whole-brain')
```

Comment:
This might be a stupid comment, but I don't understand why this is set to `whole-brain` while the function is supposed to compute the gray matter mask.
Moreover, it seems weird that, prior to this PR, this function was calling `compute_brain_mask` without any `mask_type` option, and therefore not doing what it says it should, or am I missing something? :thinking:

Reply:
This PR comes from a previous PR where the old function `compute_gray_matter_mask` was renamed to `compute_brain_mask` because it was not loading the MNI gray-matter template but the averagared whole-brain T1 MNI152 template. Afterwards, we agreed that this renamed function could be improve to actually accommodate the gray and white matter templates, too.

Reply:
OK, makes sense indeed! :+1: